Include/cpython/unicodeobject.h

```diff
@@ -247,10 +243,6 @@ PyAPI_FUNC(int) _PyUnicode_CheckConsistency(
     int check_content);
 
 /* Fast access macros */
-#define PyUnicode_WSTR_LENGTH(op) \
```

Comment:
Ditto: please document the function removal. Here, I understand that it can be replaced with PyUnicode_AsWideCharString() which provides the length in a number of wide characters.

Reply:
Oops, ignore my comment. You just moved the function.